app/models/concerns/metadatable.rb

```diff
@@ -0,0 +1,33 @@
+module Metadatable
+  extend ActiveSupport::Concern
+
+  require 'json-schema'
+  require 'fileutils'
+  require 'json' 
+
+  included do
+
+    def validate_sushi 
+      schema = load_schema
+      report = self.attributes.deep_transform_keys { |key| key.tr('_', '-') }
```

Comment:
I just learned that `deep_transform_keys` was removed in Rails 5.1 for security concerns. Might not be relevant here.